egov/egov-collectionweb/src/main/java/org/egov/collection/web/actions/receipts/BankRemittanceAction.java

```diff
@@ -219,9 +231,26 @@ public String listData() {
                 serviceCodeList.add(basm.getBusinessDetails());
             }
             final CFinancialYear financialYear = financialYearDAO.getFinancialYearById(finYearId);
+            Long dateInLongFromDate=0l;
+            Long dateInLongToDate=0l;
+
+            DateFormat formatter = new SimpleDateFormat("dd/MM/yyyy");
+            Date dateFrom;
+            Date toDates;
+            //try {
+                //dateFrom = formatter.parse(financialYear.getStartingDate().getTime());
+                //toDates = formatter.parse(financialYear.getStartingDate().toString());
+                dateInLongFromDate = financialYear.getStartingDate().getTime();
+                dateInLongToDate = financialYear.getEndingDate().getTime();
+
+
+           // } catch (ParseException e) {
+           //     // TODO Auto-generated catch block
+                //e.printStackTrace();
+            //}
             resultList = remittanceService.findCashRemittanceDetailsForServiceAndFund("", StringUtils.join(serviceCodeList, ","),
-                    StringUtils.join(fundCodeSet, ","), fromDate == null ? financialYear.getStartingDate() : fromDate,
```

Comment:
This logic is used to search based on the financial year if from date and end date is not selected. Why you have removed setting the financial year start date if from date is null? What if I search based on the financial year?